Test/spec/simd/simd_address.wast

```diff
@@ -0,0 +1,97 @@
+;; Load v128 data with different valid offset/alignment
+
+(module
+  (memory 1)
+  (data (i32.const 0) "abcdefghijklmnopqrstuvwxyz")
+  (data (offset (i32.const 65495)) "ABCDEFGHIGKLMNOPQRSTUVWXYZ")
```

Comment:
Maybe this could use hex escape codes (i.e. `\61\62\63...`) in order to better match the `v128.const` literals below?

Reply:
It might be good to use small integer hex codes to make the tests even easier to read

Reply:
Thanks, good point! I will fix it soon.

Reply:
Done.